CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ endif()
 option(run_unittests "set run_unittests to ON to run unittests (default is OFF)" OFF)
 option(run_int_tests "set run_int_tests to ON to run the integration tests (default is OFF)" OFF)
 option(use_installed_dependencies "set use_installed_dependencies to ON to use installed packages instead of building dependencies from submodules" OFF)
+option(use_cppunittest "set use_cppunittest to ON to build CppUnitTest tests on Windows (default is ON)" ON)
```

Comment:
It this really a good idea?
I remember one time unit tests were broken because we were not running straight from VS. Allowing to disable this might result in broken integration again.
What do you think?